.github/workflows/ci.yml

```diff
@@ -1,10 +1,22 @@
-name: CI CD
+name: Old CI CD
 
 on:
+  release:
+    types:
+      - published
   push:
     branches:
-      - master
-  pull_request:
+      - '*'
+    paths:
+      - '!.github/workflows/vcpkg_*'
+      - '.github/workflows/ci.yml'
+      - 'old/**'
+    tags-ignore:
+      - 'v*'
+  # pull_request:
+  #   paths:
+  #     - '!.github/workflows/vcpkg_*'
+  #     - 'old/**'
```

Comment:
Unfortunately, the `pull_request.paths` key only looks at the entire pull request list of changed files, which means it will always run on each commit if the whole PR touches any of the files. I could can it back to `pull_request` only and `push: 'master'` after we merge this monster PR... or I can do it in this PR